packages/react-core/src/components/SimpleList/examples/SimpleListBasic.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { SimpleList, SimpleListItem } from '@patternfly/react-core';
+
+export const SimpleListDemo: React.FunctionComponent = () => {
```

Comment:
Just to match the example/file name:

```suggestion
export const SimpleListBasic: React.FunctionComponent = () => {
```

Reply:
name change accepted